src/utils/constantsV3.ts

```diff
@@ -81,6 +87,7 @@ export const BALANCER_COMPOSITE_LIQUIDITY_ROUTER_NESTED: Record<
     [ChainId.SONIC]: '0xcf21664262774eBB2C2b559e13b47F6cA10F3E65',
     [ChainId.ARBITRUM_ONE]: '0x1311Fbc9F60359639174c1e7cC2032DbDb5Cc4d1',
     [ChainId.BASE]: '0xf23b4DB826DbA14c0e857029dfF076b1c0264843',
+    [ChainId.MONAD_TESTNET]: '0x36caC20dd805d128c1a6Dd16eeA845C574b5A17C',
```

Comment:
I set this as same address as boosted, which is not the case on other networks where we are using two different versions of the composite router: one for nested and one for boosted